src/olympia/amo/templates/amo/404-responsive.html

```diff
@@ -8,7 +8,7 @@
     {% endif %}
     <title>{% block title %}{{ _('Page Not Found') }}{% endblock %}</title>
     <link rel="shortcut icon" type="image/x-icon"
-          href="{{ STATIC_URL }}img/favicon.ico">
+          href="{{ STATIC_URL }}img/favicon.ico?v=2">
```

Comment:
`{{ static('img/favicon.ico') }}` will do proper cache busting, for the next time we change it.

Reply:
Ah cool, I'll update this to use that.